proto/src/determined/api/v1/checkpoint.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package determined.api.v1;
+option go_package = "github.com/determined-ai/determined/proto/pkg/apiv1";
+
+import "determined/experiment/v1/checkpoint.proto";
+
+// Get the requested checkpoint.
+message GetCheckpointRequest {
+  // The uuid for the requested checkpoint.
+  string checkpoint_uuid = 1;
+}
+
+// Response to GetCheckpointRequest.
+message GetCheckpointResponse {
+  // The requested checkpoint.
+  determined.experiment.v1.Checkpoint checkpoint = 1;
```

Comment:
just a question, do you think it makes more sense to have `determined.experiment.v1.Checkpoint checkpoint` live here and be imported in `experiment.proto` or the way it is now?

Reply:
Done